index.html

```diff
@@ -384,7 +391,7 @@ <h4>How can I register?</h4>
                     <h4>+</h4>
                 </div>
                 <div class="panel">
-                    <p>Registration for Technica is now open! You can register <a class="link" href=""><b>here</b></a>! TODO: HYPERLINK</p>
+                    <p>Registration for Technica is now open! You can register <a class="link" href="https://gotechnica.typeform.com/to/TcQ4ecwG"><b>here</b></a>! TODO: HYPERLINK</p>
```

Comment:
can you remove the "TODO: HYPERLINK" lol

Reply:
Same here lol